lighthouse-core/computed/trace-of-tab.js

```diff
@@ -1,191 +1,22 @@
 /**
- * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
no reason to update these?

Reply:
oh, I guess the moved file keeps the year and this file is "new". Blame seems to work on the new files, but I'm curious what it'll do here given that the github PR UI doesn't see it that way

Reply:
yeah as long as we don't squash I think we're OK, when you look at a long history it won't do the smart diff, but blame and individual steps should be 👌